index.html

```diff
@@ -3279,11 +3260,38 @@ <h3><dfn>Writing or pushing content</dfn></h3>
             <a>recordType</a>.
           </li>
           <li>
-            Set the |ndef|'s <a>PAYLOAD LENGTH field</a> to |length|.
+            If the type of a |record|'s <a>data</a> is a {{BufferSource}},
```

Comment:
It looks like steps 3 and 4 are the only differences between "Mapping external data to NDEF" and "Mapping local type to NDEF". Is there a way to emphasize this?

Reply:
Good observation. Let's factor out that as a parametrised algorithm in a next PR. Should we track this as an issue?

Reply:
Sure. Please open one and link to this comment.